backend/handlers/handlers.go

```diff
@@ -832,7 +832,11 @@ func (handlers *Handlers) postExportAccountSummary(_ *http.Request) (interface{}
 	if err != nil {
 		return nil, err
 	}
-	path := filepath.Join(downloadsDir, name)
+	suggestedPath := filepath.Join(downloadsDir, name)
+	path := handlers.backend.Environment().GetSaveFilename(suggestedPath)
```

Comment:
was looking for where the button is for this. turns out it's behind a `debug` in https://github.com/digitalbitbox/bitbox-wallet-app/blob/f19e1a76/frontends/web/src/routes/account/summary/accountssummary.tsx#L198-L215.